wolfcrypt/test/test.c

```diff
@@ -212,7 +212,7 @@ const byte const_byte_array[] = "A+Gd\0\0\0";
         int ret;
         char tmpBuf[80];
 
-        ret = XSNPRINTF(tmpBuf, sizeof(tmpBuf), format, args);
+        ret = vsnprintf(tmpBuf, sizeof(tmpBuf), format, args);
```

Comment:
This file needs to be portable. Please use `XVSNPRINTF`

Reply:
:heavy_check_mark: